sympy/physics/mechanics/tests/test_joint.py

```diff
@@ -0,0 +1,239 @@
+from sympy import Symbol
+from sympy.physics.vector import ReferenceFrame, Point
+from sympy.physics.mechanics import inertia, RigidBody, KanesMethod, \
+    dynamicsymbols
+
+from pydy.bodies import Body
+from pydy.joints import PinJoint, SlidingJoint, CylindricalJoint
+from pydy.joints_method import JointsMethod
+
+
+class TestPinJoint():
+    def setup(self):
+        parent = Body('parent')
+        child = Body('child')
+
+        gravity = Symbol('gravity')
+        # add gravity force in y direction at the center of mass of child.
+        child.add_force((0, child.get_mass() * gravity, 0), child.get_masscenter())
+
+        self.pin_joint = PinJoint('pin_joint', parent, child, parent_point_pos=(1, 1, 1),
+                             child_point_pos=(1, 1, 1))
```

Comment:
I think it would be preferable if you used symbols in all the tests instead of number. For example, this should be `parent_point_pos=(a, b, c)`, etc.